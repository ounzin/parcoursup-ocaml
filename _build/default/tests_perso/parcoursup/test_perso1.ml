open Test_parcoursup_utils2

let session = nouvelle_session ()

let () = 
  ajoute_candidat session ~nom_candidat:"Nabil";
  ajoute_candidat session ~nom_candidat:"Uriel";
  ajoute_formation session ~nom_formation:"Université Côte d'Azur" ~capacite:1;
  ajoute_formation session ~nom_formation:"Université de Toulon" ~capacite:1;
  ajoute_voeu session 
    ~rang_repondeur:(Some 1) 
    ~nom_candidat:"Nabil" 
    ~nom_formation:"Université Côte d'Azur";
    ajoute_voeu session 
    ~rang_repondeur:(Some 2) 
    ~nom_candidat:"Nabil" 
    ~nom_formation:"Université de Toulon";
    ajoute_voeu session 
    ~rang_repondeur:(Some 1) 
    ~nom_candidat:"Uriel" 
    ~nom_formation:"Université Côte d'Azur";
    ajoute_voeu session 
    ~rang_repondeur:(Some 2) 
    ~nom_candidat:"Uriel" 
    ~nom_formation:"Université de Toulon";
  ajoute_commission session 
    ~nom_formation:"Université Côte d'Azur" 
    ["Nabil";"Uriel"]; 
    ajoute_commission session 
    ~nom_formation:"Université de Toulon" 
    ["Nabil";"Uriel"]; 
  reunit_commissions session;
  nouveau_jour session;
  affiche_voeux_en_attente session "Nabil";
  affiche_propositions_en_attente session "Nabil";
  affiche_voeux_en_attente session "Uriel";
  affiche_propositions_en_attente session "Uriel";
  renonce session  ~nom_candidat:"Nabil"  ~nom_formation:"Université de Toulon";
  renonce session  ~nom_candidat:"Nabil"  ~nom_formation:"Université Côte d'Azur";
  renonce session  ~nom_candidat:"Uriel"  ~nom_formation:"Université de Toulon";
  renonce session  ~nom_candidat:"Uriel"  ~nom_formation:"Université Côte d'Azur";
  nouveau_jour session;
  affiche_voeux_en_attente session "Nabil";
  affiche_propositions_en_attente session "Nabil";
  affiche_voeux_en_attente session "Uriel";
  affiche_propositions_en_attente session "Uriel";