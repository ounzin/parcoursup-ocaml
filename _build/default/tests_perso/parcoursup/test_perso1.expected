nouvelle session
ajoute candidat Nabil
ajoute candidat Uriel
ajoute formation Université Côte d'Azur (1 places)
ajoute formation Université de Toulon (1 places)
ajoute voeu de Nabil pour Université Côte d'Azur (rang=1)
ajoute voeu de Nabil pour Université de Toulon (rang=2)
ajoute voeu de Uriel pour Université Côte d'Azur (rang=1)
ajoute voeu de Uriel pour Université de Toulon (rang=2)
commission de classement de Université Côte d'Azur : [Nabil; Uriel]
commission de classement de Université de Toulon : [Nabil; Uriel]
réunit commissions
nouveau jour
Voeux en attente de Nabil : {}
Propositions en attente de Nabil : {Université Côte d'Azur}
Voeux en attente de Uriel : {Université Côte d'Azur(0)}
Propositions en attente de Uriel : {Université de Toulon}
Nabil renonce à Université de Toulon
Nabil renonce à Université Côte d'Azur
Uriel renonce à Université de Toulon
Uriel renonce à Université Côte d'Azur
nouveau jour
Voeux en attente de Nabil : {}
Propositions en attente de Nabil : {}
Voeux en attente de Uriel : {}
Propositions en attente de Uriel : {}
